app/services/analytics_events.rb

```diff
@@ -565,6 +565,7 @@ def idv_verify_in_person_troubleshooting_option_clicked(flow_path:, **extra)
     )
   end
 
+  # @param [Hash] variant The IPP CTA Variant that was shown to the user
   # The user was shown a variant of the IPP CTA
   def idv_ipp_cta_variant(variant:, **extra)
     preserve_testing_value({ ipp_cta_variant: variant })
```

Comment:
I can't leave a comment on the line... but the dynamic string as the event name is the exact opposite of what we want in this file. It's difficult to find and aggregate across the variants, it would be much more effective to have a static string  + an explicit `variant:` property so that we can filger on these in Cloudwatch, I would strongly recommend changing this to:

```ruby
    track_event(
      'IdV: IPP CTA Variant Displayed,
      variant: variant,
      **extra,
    )
```

Reply:
nevermind was looking at the wrong part of the diff, gonna delete + redo this comment